xml/templates/message_templates/contribution_invoice_receipt_html.tpl

```diff
@@ -143,6 +143,7 @@
                 <td colspan = "3"></td>
                 <td style = "padding-left:20px;text-align:right;"><b><font size = "1">{ts 1=$defaultCurrency}TOTAL %1{/ts}</font></b></td>
                 <td style = "padding-left:34px;text-align:right;"><font size = "1">{$amount|crmMoney:$currency}</font></td>
+                <td style = "padding-left:34px;"><font size = "1" align = "right"></fonts></td>
```

Comment:
in this place & 2 others in the tpl (which you probably copied one of) the opening tag is font & the closing tag is fonts. We should merge this first & then fix that